source/daplink/drag-n-drop/flash_manager.c

```diff
@@ -95,6 +95,10 @@ error_t flash_manager_init(const flash_intf_t *flash_intf)
         return status;
     }
 
+#ifdef FLASH_PARTIAL_ERASE
+    // Processing for low-speed devices of chip erasure.
+    flash_manager_printf("    skip intf->erase_chip\r\n");
+#else
```

Comment:
FLASH_PARTIAL_ERASE is a very useful feature. This would be nice as a run-time configurable value. Maybe a function to set the mode?
flash_manager_init(const flash_intf_t *flash_intf, flash_mode_t)
or maybe a seperate function
flash_manager_mode(CHIP_ERASE) / flash_manager_mode(SECTOR_ERASE)

What do you think @d-kato?